src/azurelinux/3.0/helix/Dockerfile

```diff
@@ -29,6 +30,7 @@ RUN tdnf install --setopt tsflags=nodocs --refresh -y \
         python3 \
         python3-pip \
         shadow-utils \
+        sudo \
```

Comment:
What requires `sudo`? These images have been working for a long time without it.

Reply:
`sudo` worked in the majority of the Helix containers and some teams use it in their testing scenarios. I also kept hitting issues in the updated containers as I tried to understand problems while I worked through iterations of this PR. so, consistency, muscle memory, and customer scenarios all play into this

we can discuss removing `sudo` where that's an option later. doing so will make some things more difficult around investigations and enhancements in this repo e.g., on OSes whose package managers can only be run as root &mdash; even when querying but it may not break the world